db/seeds.rb

```diff
@@ -1,11 +1,31 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
-# require 'csv'
-
-# csv_text = File.read(Rails.root.join('lib', 'seeds', 'real_estate_transactions.csv'))
-# puts csv_text
\ No newline at end of file
+require 'csv'
+
```

Comment:
Can we wrap this file to create the rails csv_load:all task?

```
namespace :csv_load do

desc "Import customers from CSV"
task customers: :environment do
  # CSV code here 
    end
  end
end
```

Reply:
@koshea9 yes, I think this is our next step so we can get those custom rake commands going